packages/app-studio-toolkit/package.json

```diff
@@ -28,7 +28,7 @@
     "package": "node ./scripts/package-vsix.js"
   },
   "dependencies": {
-    "@sap/artifact-management": "1.17.2",
+    "@sap/artifact-management": "1.18.0",
```

Comment:
should not the `artifact-managment-types` be also updated 
in `app-studio-types` package?

Reply:
Thanks for your reminder. I will update artifact-managment-types to 1.18.0.